devices/dstt.cpp

```diff
@@ -92,7 +92,7 @@ Known flashchips that are "unsupported":
     0xDAC2 "MACRONIX MX29LV800T"
 */
 
-#include "device.h"
+#include "../device.h"
```

Comment:
relative paths = bad

Reply:
Actually, here's a case for relative paths: we don't know how projects are going to use this. Not using relative paths forces them to add this as an include directory, while using relative paths ensures that it works.